docker/app/Dockerfile

```diff
@@ -12,6 +12,15 @@ RUN wget -q -O /tmp/shellcheck.tar.xz "https://storage.googleapis.com/shellcheck
   && rm /tmp/shellcheck.tar.xz \
   && shellcheck --version
 
+RUN wget -q -O get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 \
+  && chmod 700 get_helm.sh \
+  && ./get_helm.sh \  
```

Comment:
Downloading the `get_helm.sh` script means we should also clean it up after use (see comment below). Alternatively, we can also wget straight into bash with a pipe to simplify things further. Something like this should work:

```sh
wget -q -O- https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
```